front/src/store/index.js

```diff
@@ -46,6 +46,34 @@ export default new Vuex.Store({
     },
     logout ({ commit }) {
       commit('CLEAR_USER_DATA');
+    },
+    recoveryPasswordSendEmail (context, payload) {
+      return new Promise((resolve, reject) => {
+        axios.post(`${serverPath}api/ForgotPassword`,
+          { Email: payload.email }).then(resp => {
+          if (resp.status !== 200) {
+            reject(resp.data);
+          } else {
+            localStorage.setItem('email', payload.email);
+            resolve();
+          }
+        });
+      });
+    },
+    recoveryPasswordSendPassword (context, passwordandtoken) {
+      return new Promise((resolve, reject) => {
+        axios.post(`${serverPath}api/ForgotPassword/ResetPassword`,
+          { Email: localStorage.getItem('email'), Password: passwordandtoken.password }, {
+            headers: { Authorization: 'Bearer ' + passwordandtoken.token }
+          }).then(resp => {
+          if (resp.data !== '') {
+            localStorage.removeItem('email');
```

Comment:
making a mutation for such limited use doesn't seem to make sense. or is there?

Reply:
why it doesn't have sense?

Reply:
because it is one line?

Reply:
I'm not sure that anyone else will use mail in this way

Reply:
it is not only for reusing. it is about approach to storing data. we use vuex. we use flux architecture. all application data is stored in store and managed via store. If you delete something or add not via store in could make bugs. because some things could be higly connected with some information or actions chain. it is event based architecture. and when something happens with common application data not via main data management way - it is really not good. Please, pay more attention on articles and my message in education channel. Read articles about vuex and flux